docs/application/native/guides/internationalization/i18n.md

```diff
@@ -779,15 +779,15 @@ The UCharIter functions return code unit values in the range of 0 - 0xffff. Befo
 
 
 <a name="plural_format"></a>
-## Format plural messages with PluralFormat
+## Format Plural Messages with PluralFormat
 
 The PluralFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) applications) allows you to [format defined plural words](#plural_format_examples). Alongside with PluralRules module, you can define rules to which format would apply.
 
 PluralFormat supports creation of locale-dependent messages with plural inflection. To enable plural formatting,
 you have to specify each message for every plural case that will be used, based on the desired language.
 Then, the PluralFormat module will select appropriate text based on the provided number.
 
-### Issues of plural forms in internationalized messages
+### Issues of Plural Forms in Internationalized Messages
 
 Many languages have different approach to create plural nouns. The locale-dependent message is workable when the framework
```

Comment:
Please incorporate the following comments:

Line 817: The heading is not changed to title case.
Line  945 : please change “2” to “two”